client/src/components/Onboarding/OnboardingAdmin.jsx

```diff
@@ -97,6 +97,25 @@ const OnboardingResetLinks = ({ callback, disabled, disableSkip, action }) => {
         )}
         Skip
       </Button>
+      {forceStorageCreation && ( 
+	<>
+          <>|</>
+          <Button
+            color="link"
+            className={styles['action-link']}
+            onClick={() => callback('force_create_storage')}
+            disabled={disabled}
+          >
+            {action === 'force_create_storage' && (
+              <LoadingSpinner
+                placement="inline"
+                className="onboarding-admin__action-spinner"
+              />
+            )}
+          Force Create
```

Comment:
Should this be `Force Recreate` perhaps?

Reply:
Sounds good!